ajax/libs/videojs-overlay/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "videojs-overlay",
+  "description": "A plugin to display simple overlays during video playback.",
+  "author": "Brightcove, Inc.",
+  "license": "Apache-2.0",
+  "version": "1.1.4",
```

Comment:
as CI said, let's remove `"version": "1.1.4",`.
We can add this library via a [single package.json](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#e-adding-a-new-library-by-a-single-packagejson)
So we don't need the `version` field.